lib/tasks/helpers/version_bump.rb

```diff
@@ -46,7 +46,7 @@ def self.write_file(path, file)
   def self.determine_bump_type
     file = read_file('CHANGELOG.md')
     lines = file.split('## ')[1].split('- **')
-    return MAJOR if lines.first.include?('Major version')
+    return MAJOR if lines.any? { |line| line.include?('Breaking Change:') }
```

Comment:
This makes sense! I guess we would never do a major if there wasn't a breaking change, so presumably all Majors would have this. Nice